### ISSUE TITLE
Block inspector behaves erratically when using multiple editors

### ISSUE THREAD

**Issue by vonloxx** [Automattic/isolated-block-editor#220]
When using multiple editors, the block inspector component behaves erratically:
![block-inspector-1](https://user-images.githubusercontent.com/1254357/229631381-f88b0f38-c53a-49a0-8403-bc5b5adb1354.gif)

**Comment by johngodley** [comment]
You will need to design your system so this cross over is avoided.

**Comment by vonloxx** [comment]
ouch 😓 But it happens because the toggle actions uses the same interface store, maybe?

**Comment by johngodley** [comment]
Undoubtedly it is because of shared state. However, it is difficult to fully separate this as Gutenberg is just not designed for that kind of thing. Attempts have been made and some of these used to work well, but changes to Gutenberg along the way have meant that some things do bleed through. Is your example above a real thing or just theoretical?

**Comment by vonloxx** [comment]
Yep, I understand that. The examples above are taken from this repository storybook (Multiple editors) with inspector enabled.
Anyway, I'll try to come up with something to overcome, at least this issue.

**Comment by johngodley** [comment]
Ah ok, yes. It's a moving target so things will change. Hopefully full isolation can be brought back, but currently it's tricky.

**Comment by hwkdev** [comment]
Hello!

I can report the same issue when using multiple instances of the Isolated Block Editor on the same page (within WordPress).

Did you find any workaround @vonloxx?

Thanks!

**Comment by vonloxx** [comment]
@hawkidoki Actually I ended using the block editor component (wrapped on my own components) since a lot of work was done so the block editor can work in a more "isolated" way. More info at https://github.com/WordPress/gutenberg/issues/53874
